data/pathfinder/paizo/roleplaying_game/advanced_players_guide/apg_abilities_companion.lst

```diff
@@ -77,7 +77,7 @@ Pincers							KEY:Evolution ~ Pincers								CATEGORY:Special Ability	TYPE:Evolu
 Pull								KEY:Evolution ~ Pull								CATEGORY:Special Ability	TYPE:EvolutionChoice.Extraordinary																																																											PREMULT:1,[PREREACHGTEQ:10],[PREABILITY:1,CATEGORY=Special Ability,Evolution ~ Reach]																																																																																					DESC:The eidolon gains the ability to pull creatures closer with a successful attack. Whenever the eidolon makes a successful attack of the selected type, it can attempt a free combat maneuver check. If successful, the target of the attack is pulled 5 feet closer to the eidolon. This ability only works on creatures of a size equal to or smaller than the eidolon. Creatures pulled in this way do not provoke attacks of opportunity.																																											MULT:YES	CHOOSE:WEAPONPROFICIENCY|PC,TYPE=Natural																						ABILITY:Special Ability|AUTOMATIC|Pull
 Push								KEY:Evolution ~ Push								CATEGORY:Special Ability	TYPE:EvolutionChoice.Extraordinary																																																																																																																																																														DESC:The eidolon gains the ability to push creatures away with a successful attack. Whenever the eidolon makes a successful attack of the selected type, it can attempt a free combat maneuver check. If successful, the target of the attack is pushed 5 feet directly away from the eidolon. This ability only works on creatures of a size equal to or smaller than the eidolon. Creatures pushed in this way do not provoke attacks of opportunity.																																										MULT:YES	CHOOSE:WEAPONPROFICIENCY|PC,TYPE=Natural																						ABILITY:Special Ability|AUTOMATIC|Push
-Reach								KEY:Evolution ~ Reach								CATEGORY:Special Ability	TYPE:EvolutionChoice.SpecialQuality.Extraordinary.EvolutionSerpentine																																				!PREABILITY:1,CATEGORY=Special Ability,Evolution ~ Reach																																																																																																											DESC:Reach increased 5 ft. with your %1|%LIST																																																																																																												MULT:YES	CHOOSE:USERINPUT|TITLE=Attack
+Reach								KEY:Evolution ~ Reach								CATEGORY:Special Ability	TYPE:EvolutionChoice.SpecialQuality.Extraordinary.EvolutionSerpentine																																				!PREABILITY:1,CATEGORY=Special Ability,Evolution ~ Reach																																																																																																											DESC:Reach increased 5 ft. with your %1|%LIST																																																																																																												MULT:YES	CHOOSE:USERINPUT|TITLE=Enter Attack Type		BONUS:WEAPONPROF=%LIST|REACH|REACH+5
```

Comment:
Very poor method to use, and I'd be surprised if this really works. I'd suggest a more smart CHOOSE with both Natural and PC so the choice is limited to attacks that should benefit. Also, REACH+5 is not the best method, a simple 5 will suffice.